src/utils/hints/stateChannelsHints.js

```diff
@@ -12,4 +12,30 @@ export const stateChannelsHints = {
   initiator: 'The account that initiated the creation of the state channel and covered the fees for its creation.',
   responder: 'The participating account that joined the state channel.',
   lastKnownRound: 'Participants can for example protect themselves from being cheated on by providing a snapshot from the latest off-chain round to the chain.',
+  slashedRound: 'The round in which the slash was triggered.',
+  slashedBy: 'The account that slashed (disputed) malicious ChannelCloseSoloTx or ChannelForceProgressTx.',
+  initiatorCreateAmount: 'Unsigned amount of coins the initiator commits to the channel.',
+  initiatorCloseAmount: 'Final balance for the initiator.',
+  initiatorSettleAmount: 'Unsigned amount of coins the initiator gets from the channel.',
+  responderCreateAmount: 'Unsigned amount of coins the responder commits to the channel.',
+  responderCloseAmount: 'Final balance for the responder.',
+  responderSettleAmount: 'Unsigned amount of coins the responder gets from the channel.',
+  settledBy: 'Participant of the channel that posts the settling transaction.',
+  snapshotRound: 'Channel\'s next round.',
+  snapshottedBy: 'The account that initated the snapshot of the state channel.',
+  forceProgressRound: 'The round in which the state channel was forced to progress.',
+  forcedBy: 'Participant or a delegate of the channel that posts the force progress transaction.',
+  depositor: 'Account that deposited funds to the state channel.',
+  depositRound: 'The channel\'s internal round that applies the deposit.',
+  depositAmount: 'Amount deposited to the state channels.',
+  withdrawRound: ' The channel\'s internal round that applies the withdraw.',
+  withdrawAmount: 'Amount withdrawn from the state channel.',
+  withdrawRecipient: 'Account that received the withdrawn funds.',
+  initiatorDelegates: 'List of delegate account ids by initiator. The delegates play a role in the solo closing sequence.',
+  responderDelegates: 'List of delegate account ids by responder. The delegates play a role in the solo closing sequence.',
+  lockPeriod: 'Period for disputes after solo operations.',
+  closeRound: 'The round in which the state channel was proposed to be closed.',
+  closeProposedBy: 'The participant of the channel that posts the closing transaction.',
+  sentBy: 'State channel participant or delegate that posts the slashing transaction.',
+  channelReserve: 'The minimum amount of AE coins that must remain locked in the channel.',
```

Comment:
```suggestion
  channelReserve: 'The minimum amount of AE coins that must remain locked in any of the off-chain accounts in the channel.',
```
It is not the total amount locked in the channel but a minimum threshold for both accounts independently

Reply:
thanks, accepted